src/NuGet.Core/NuGet.DependencyResolver.Core/Remote/RemoteDependencyWalker.cs

```diff
@@ -52,7 +52,9 @@ public async Task<GraphNode<RemoteResolveResult>> WalkAsync(LibraryRange library
             RuntimeGraph runtimeGraph,
             Func<LibraryRange, (DependencyResult dependencyResult, LibraryDependency conflictingDependency)> predicate,
             GraphEdge<RemoteResolveResult> outerEdge,
-            TransitiveCentralPackageVersions transitiveCentralPackageVersions)
+            TransitiveCentralPackageVersions transitiveCentralPackageVersions,
+            LibraryIncludeFlags inheritedIncludeFlags = LibraryIncludeFlags.All,
+            LibraryIncludeFlags inheritedSuppressParent = LibraryIncludeFlags.None)
```

Comment:
I'm not sure you need the suppressParent here? 
There is not, it's include flags only.

Reply:
Updated the PR to exclude the SuppressParent